client/src/elm/Translate.elm

```diff
@@ -8987,12 +8992,12 @@ translationSet trans =
 
                 EducationLowBackPain ->
                     if isInitial then
-                        { english = "Low back pain in pregancy - provided health education on " ++ date
+                        { english = "Lower backer pain in pregancy - provided health education on " ++ date
```

Comment:
@anvmn This sentence does not seems correct. Maybe `Lower back pain in pregnancy`?